docroot/themes/humsci/humsci_basic/src/js/navigation/secondary-toggler.js

```diff
@@ -0,0 +1,31 @@
+import changeNav from './change-nav';
+
+const togglers = document.querySelectorAll('.hb-secondary-toggler');
+
+if (togglers) {
+  for (let i = 0; i < togglers.length; i += 1) {
+    const toggler = togglers[i];
+    const togglerID = toggler.getAttribute('id');
+    const togglerContent = document.querySelector("[aria-labelledby=\"".concat(togglerID, "\"]"));
+    const togglerParent = toggler.parentNode;
+    const activeTrail = togglerParent.classList.contains('hb-secondary-nav__item--active-trail');
+
+    // Togglers should always have content but in the event that they don't we
+    // don't want the rest of the togglers on the page to break.
+    if (!togglerContent) {
+      continue;
+    }
+
+    if (!activeTrail) {
+      toggler.setAttribute('aria-expanded', false);
+      togglerContent.setAttribute('aria-hidden', true);
```

Comment:
Should we replace this with `changeNav(toggler, togglerContent, false);`? We don't really changeNav in the inline JS but I think it makes sense here since we are already importing it.